src/towncrier/_project.py

```diff
@@ -67,6 +69,10 @@ def get_version(package_dir: str, package: str) -> str:
     if version := _get_metadata_version(package):
         return version
 
+    with contextlib.suppress(PackageNotFoundError):
```

Comment:
Do we need this code?

What does it to, on top of the existing `_get_metadata_version` ?

Maybe move this to `_get_metadata_version`

If this needs to stay here, then it needs a comment to explain why `_get_metadata_version` is not good enough here and why we get the version again via `importlib.metadata.

Reply:
I think that `_get_metadata_version` should do everything.

So the code should be moved there.

Maybe also remove `importlib.metadata.packages_distributions` and replace it with the current method.

Do we need both methods? Just asking.

I am not very familiar with the `importlib` API  ... but as long as the tests pass, we should be ok.

Reply:
Ok, I thought there might be some reason why it was done this way.

I replaced it as you suggested.